labellab-client/src/components/navbar/project.js

```diff
@@ -17,14 +17,23 @@ import { fetchUser } from '../../actions/index'
 import { TOKEN_TYPE } from '../../constants/index'
 import './css/navbar.css'
 
-const initialState = { isLoading: false, results: [], value: '' }
+const initialState = { isLoading: false, results: [], value: '', resultSelected: false, imageId: '' }
 
-const resultRenderer = ({ imageName, createdAt }) =>
+const resultRenderer = ({
+  imageName,
+  project,
+  _id,
+  createdAt }) =>
   (
-    <>
+    <a href={
+      '/project/' +
+      project +
+      '/images/#' +
+      _id
+    }>
       <Label content={imageName} />
       <Label content={new Date(createdAt).toISOString().substring(0, 10)} />
-    </>
+    </a>
```

Comment:
@carpecodeum Can I know the reason why you can't user `history.push` instead of the above piece of code ?

Reply:
No reason. I will make the required changes.